down-the-stack/dk_template/memory.x

```diff
@@ -0,0 +1,5 @@
+MEMORY
```

Comment:
Worth a note here as to which chip we're running on, and a link to the Manufacturer's data sheet / product page, so you can easily verify that these numbers are correct.